src/main/java/com/uber/cadence/internal/worker/SingleWorkerOptions.java

```diff
@@ -29,8 +29,15 @@
 
 public final class SingleWorkerOptions {
 
-  public static final class Builder {
+  public static Builder newBuilder() {
+    return new Builder();
```

Comment:
then the constructor defined below must be private

Reply:
Good catch. Thanks!